src/parse.c

```diff
@@ -952,19 +958,19 @@ StorageClass Parser::appendStorageClass(StorageClass storageClass, StorageClass
     {
         StorageClass u = storageClass & (STCconst | STCimmutable | STCmanifest);
         if (u & (u - 1))
-            error("conflicting attribute '%s'", Token::toChars(token.value));
+            error("conflicting attribute '%s'", Token::toChars(token.value), deprec);
     }
     if (stc & (STCgshared | STCshared | STCtls))
     {
         StorageClass u = storageClass & (STCgshared | STCshared | STCtls);
         if (u & (u - 1))
-            error("conflicting attribute '%s'", Token::toChars(token.value));
+            error("conflicting attribute '%s'", Token::toChars(token.value), deprec);
     }
     if (stc & (STCsafe | STCsystem | STCtrusted))
     {
         StorageClass u = storageClass & (STCsafe | STCsystem | STCtrusted);
         if (u & (u - 1))
-            error("conflicting attribute '@%s'", token.toChars());
+            error("conflicting attribute '@%s'", token.toChars(), deprec);
```

Comment:
No, I mean you're passing the `deprec` variable to error without using it in the format string.